DEVELOPMENT.md

```diff
@@ -0,0 +1,496 @@
+# KCP-Edge Contributor Development Environment
```

Comment:
I think the title needs adjusting; we are no longer talking about a distinct "environment" for developers, right?

Reply:
yes, we'll still have separated instructions for users versus developers, aren't we?  please, clarify. thanks!

Reply:
How about simply "Contributor Instructions"?

Reply:
"Users" may do 'development' too --- development of higher level code.  That is why I use the terms "contributor" for developers of this package and "users".

Reply:
Fixed